cmd/api/src/model/audit.go

```diff
@@ -156,6 +157,27 @@ type AuditEntry struct {
 	ErrorMsg string
 }
 
+// Necessary function for testing. Ensures all fields except CommitID match so mocking checks pass.
```

Comment:
If this is only used in tests, we should probably define it with the tests (can't easily add it as a method, but a method is just a function where the first parameter is the struct you're operating on)

Reply:
This function and the `String()` function are needed to satisfy the `Matcher` interface in gomock. It needs to exist on the type to override the default match behavior, which fails when comparing the UUID. I used [this tutorial](https://medium.com/modanisa-engineering/writing-a-custom-matcher-for-testing-with-gomock-d3ef5f13db82) as a reference.

Reply:
Ah, that makes sense. Yeah, since we don't have a way to know what the UUID is, since it's generated on the fly, I get why you'd go this route. I'm still not a big fan of having code specific to mocks in our application code, but I'm not sure what I'd recommend instead.

Side note: if you wouldn't mind changing `interface{}` to `any`, it will meet our styleguide (`any` is functionally `interface{}` so it won't break the gomock caller

Reply:
Will do! Also FWIW, I do agree on the co-mingling of test code + functional code, but this does solve the testing problem.

Reply:
Agreed, this is a pragmatic solution for now. It sounds like changing our testing to avoid it would be more work, and this is already a significant amount of work as is. We can clean it up in the future when a better idea comes along.